tests/12_radioactive_decay/ref_output/decay_vtk/mass_balance.txt

```diff
@@ -18,362 +18,362 @@
 9.5	"ALL"	"A"	-4.02456e-18	0	-4.02456e-18	0.135	0	0	0	-2.01228e-18	0	-3.82333e-17	0	2.77556e-17
-9.5	"ALL"	"B"	-4.214e-17	0	-4.214e-17	1.41355	0	0	0	-2.07111e-17	0.0240772	-2.69408e-16	1.14355	2.22045e-16
+9.5	"ALL"	"B"	-4.23611e-17	0	-4.23611e-17	1.42096	0	0	0	-2.08314e-17	0.0234227	-2.72462e-16	1.15096	2.22045e-16
 9.5	"ALL"	"C"	-1.20737e-17	0	-1.20737e-17	0.405	0	0	0	-6.03684e-18	0	-1.147e-16	0	1.11022e-16
-9.5	"ALL"	"D"	-1.15575e-18	0	-1.15575e-18	0.0387686	0	0	0	-6.63805e-19	-0.00576482	-5.77155e-17	-0.501231	5.55112e-17
+9.5	"ALL"	"D"	-1.15575e-18	0	-1.15575e-18	0.0387685	0	0	0	-6.63803e-19	-0.00576482	-5.77155e-17	-0.501232	1.38778e-16
 9.5	"ALL"	"E"	-2.01228e-17	0	-2.01228e-17	0.675	0	0	0	-1.00614e-17	0	-1.91167e-16	0	2.22045e-16
-9.5	"ALL"	"F"	-4.99893e-18	0	-4.99893e-18	0.167684	0	0	0	-2.77243e-18	-0.0183124	-1.31677e-16	-0.642316	2.498e-16
+9.5	"ALL"	"F"	-4.77783e-18	0	-4.77783e-18	0.160268	0	0	0	-2.65212e-18	-0.0176579	-1.28622e-16	-0.649732	8.32667e-17
 10	"region_1"	"A"	0	0	0	0.135	0	0	0	0	0	0	0	0
-10	"region_1"	"B"	0	0	0	1.43525	0	0	0	0	0	0	0	0
+10	"region_1"	"B"	0	0	0	1.44205	0	0	0	0	0	0	0	0
 10	"region_1"	"C"	0	0	0	0.405	0	0	0	0	0	0	0	0
-10	"region_1"	"D"	0	0	0	0.03375	0	0	0	0	0	0	0	0
+10	"region_1"	"D"	0	0	0	0.0337499	0	0	0	0	0	0	0	0
 10	"region_1"	"E"	0	0	0	0.675	0	0	0	0	0	0	0	0
-10	"region_1"	"F"	0	0	0	0.150996	0	0	0	0	0	0	0	0
+10	"region_1"	"F"	0	0	0	0.1442	0	0	0	0	0	0	0	0
 10	"IMPLICIT BOUNDARY"	"A"	-4.02456e-18	0	-4.02456e-18	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"B"	-4.27871e-17	0	-4.27871e-17	0	0	0	0	0	0	0	0	0
+10	"IMPLICIT BOUNDARY"	"B"	-4.29897e-17	0	-4.29897e-17	0	0	0	0	0	0	0	0	0
 10	"IMPLICIT BOUNDARY"	"C"	-1.20737e-17	0	-1.20737e-17	0	0	0	0	0	0	0	0	0
 10	"IMPLICIT BOUNDARY"	"D"	-1.00614e-18	0	-1.00614e-18	0	0	0	0	0	0	0	0	0
 10	"IMPLICIT BOUNDARY"	"E"	-2.01228e-17	0	-2.01228e-17	0	0	0	0	0	0	0	0	0
-10	"IMPLICIT BOUNDARY"	"F"	-4.50144e-18	0	-4.50144e-18	0	0	0	0	0	0	0	0	0
+10	"IMPLICIT BOUNDARY"	"F"	-4.29882e-18	0	-4.29882e-18	0	0	0	0	0	0	0	0	0
 10	"ALL"	"A"	-4.02456e-18	0	-4.02456e-18	0.135	0	0	0	-2.01228e-18	0	-4.02456e-17	0	2.77556e-17
-10	"ALL"	"B"	-4.27871e-17	0	-4.27871e-17	1.43525	0	0	0	-2.107e-17	0.0217067	-2.90478e-16	1.16525	2.22045e-16
+10	"ALL"	"B"	-4.29897e-17	0	-4.29897e-17	1.44205	0	0	0	-2.11806e-17	0.0210864	-2.93642e-16	1.17205	2.22045e-16
 10	"ALL"	"C"	-1.20737e-17	0	-1.20737e-17	0.405	0	0	0	-6.03684e-18	0	-1.20737e-16	0	1.11022e-16
-10	"ALL"	"D"	-1.00614e-18	0	-1.00614e-18	0.03375	0	0	0	-5.77875e-19	-0.00501857	-5.82934e-17	-0.50625	1.11022e-16
+10	"ALL"	"D"	-1.00614e-18	0	-1.00614e-18	0.0337499	0	0	0	-5.77874e-19	-0.00501856	-5.82934e-17	-0.50625	1.17961e-16
 10	"ALL"	"E"	-2.01228e-17	0	-2.01228e-17	0.675	0	0	0	-1.00614e-17	0	-2.01228e-16	0	2.22045e-16
-10	"ALL"	"F"	-4.50144e-18	0	-4.50144e-18	0.150996	0	0	0	-2.49947e-18	-0.0166881	-1.34176e-16	-0.659004	2.22045e-16
+10	"ALL"	"F"	-4.29882e-18	0	-4.29882e-18	0.1442	0	0	0	-2.38891e-18	-0.0160678	-1.31011e-16	-0.6658	8.32667e-17
```

Comment:
Again, Pade approximant gives more precise results (almost the same as with time step 10 times smaller).